debian/control

```diff
@@ -14,23 +14,35 @@ Build-Depends: debhelper (>= 10),
           libncursesw5-dev,
           libdaemon-dev,
           libjansson-dev,
-          python3-pip
+          python3-pip,
+          dh-python,
```

Comment:
Is `dh-python` required for the execution or just for the construction? I am asking to be sure that the dependency is in the right place

Reply:
Only for construction. `dh-python` is the Debian helper utility for Python